xbanish.c

```diff
@@ -209,6 +212,9 @@ main(int argc, char *argv[])
 			switch (xie->evtype) {
 			case XI_RawMotion:
 			case XI_RawButtonPress:
+				if (ignore_scroll && ((xie->detail > 3 && xie->detail < 8) ||
```

Comment:
I'd prefer `detail >= 4` and `detail <= 7` to make it clearer which button ids this includes.